scripts/cluster-scanner/internal/search_clusters.go

```diff
@@ -0,0 +1,16 @@
+package internal
+
+import (
+	"log/slog"
+
+	"github.com/spectrocloud/palette-sdk-go/api/models"
+	"github.com/spectrocloud/palette-sdk-go/client"
+)
+
+func SearchClusters (paletteClient *client.V1Client, logger *slog.Logger) ([]*models.V1SpectroClusterSummary, error) {
```

Comment:
You should consider adding doc comments explaining what all these exported functions do. https://tip.golang.org/doc/comment#func
Speaking of testing, since this function simply wraps around the external client, I wouldn't bother testing it. Testing `SearchOldClusters` and the formatting should suffice.

Reply:
Added the comments :D